server/app.js

```diff
@@ -0,0 +1,21 @@
+import '@babel/polyfill';
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from'cors';
+import db from './database/models/index';
+
+
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+// support parsing of application/x-www-form-urlencoded post data
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
You don't need to install `bodyParser` for this. You can use `express`. Look at using [express.urlencoded](https://expressjs.com/en/api.html#express.urlencoded)